integration-tests/sci-embedding.spec.js

```diff
@@ -0,0 +1,83 @@
+'use strict'
+
+const {
+  FakeAgent,
+  spawnProc,
+  createSandbox,
+  curlAndAssertMessage
+} = require('./helpers')
+const path = require('path')
+const { assert } = require('chai')
+const { SCI_COMMIT_SHA, SCI_REPOSITORY_URL } = require('../packages/dd-trace/src/constants')
+
+const DUMMY_GIT_SHA = '13851f2b092e97acebab1b73f6c0e7818e795b50'
+const DUMMY_REPOSITORY_URL = 'git@github.com:DataDog/sci_git_example.git'
+
+describe('sci embedding', function () {
```

Comment:
Does this need to be an integration test?

Reply:
I thought of integration because of the usage of "external" env vars, but this is done in other unit tests. I'll migrate 👍